engine/src/main/scala/cromwell/engine/workflow/ValidateActor.scala

```diff
@@ -78,7 +65,7 @@ class ValidateActor(wdlSource: WdlSource, workflowInputs: Option[WdlJson], workf
     * 2.) Raw inputs and it's coercion
     * 3.) Workflow Options
     * 4.) RuntimeAttributes w.r.t. a backend
-    * @return
+    * @return The result of the validation as instance of `ValidationResult`
     */
   private def validateAll(): Future[ValidationResult] = {
```

Comment:
@mcovarr Can you explain how that is more standardized? I think this reads better: 

```scala
validateAll() map { validationResult =>
        requester ! validationResult
        context.stop(self)
      }
```

with `validateAll()` remaining the same. To me, just for sending the response and stopping itself doesn't warrant a new function with only side effects